modules/tinymce/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Selecting a figure with `class="image"` would incorrectly highlight the link toolbar button #TINY-8832
 - Specifying a single, non-default list style for the `advlist_bullet_styles` and `advlist_number_styles` options was not respected. #TINY-8721
 - Fixed various issues that occurred when formatting `contenteditable` elements. #TINY-8905
+- Link url field would sometimes allow users to add spaces when a title was set #TINY-8775
```

Comment:
I don't think we typically use the word 'users'. Maybe something like this?
```suggestion
- Spaces could be incorrectly added to the Link plugin url field in certain cases #TINY-8775
```

Reply:
```suggestion
- Urlinput dialog components would sometimes incorrectly allow spaces to be added before or after the URL #TINY-8775
```
I think this would be better, as this is about the whole `urlinput` component not just the link plugin specifically.

Reply:
Or alternatively trying to use Luke's wording:
```suggestion
- Spaces could be incorrectly added to urlinput dialog components in certain cases #TINY-8775
```